components/css/newsletter-text-only/_contentPage.scss

```diff
@@ -2,194 +2,227 @@ $white:#ffffff;
-			font-size: 1.4em;
-			line-height: 1.3em;
-			padding: 1em 0 0 0;
-				&-light {
-					color: #000000;
-				}
-		}
-	
-		&__subtitle {
-			color: $light-gray;
-			font-size: 1em;
-			line-height: 1.5em;
-			padding-top: .5em;
-			padding-bottom: .8em;
-		}
+	&__title {
+		color: $color-campusGold;
+		font-size: 1.4em;
+		line-height: 1.3em;
+		padding: 1em 0 0 0;
+			&-light {
+			color: $black;
+			}
+	}
 	
-		&__teaser {
-			color: $white;
-			font-size: .9em;
-			line-height: 1.5em;
-			padding-bottom: .5em;
-		}
+	&__subtitle {
+		color: $light-gray;
+		font-size: 1em;
+		line-height: 1.5em;
+		padding-top: .5em;
+		padding-bottom: .8em;
+	}
 	
-		&--light {
-			background-color:$white;	
-		}
+	&__teaser {
+		color: $white;
+		font-size: .9em;
+		line-height: 1.5em;
+		padding-bottom: .5em;
+	}
 	
-		&__link {
-			font-size: .9em;
-			line-height: 1.4em;
-				&-other {   
-					display:flex;
-					align-items:center;
-					margin-top: 20px;
-        
-				&:link,&:visited,&:hover,&:active{
-					color:$campusGold;
-					text-decoration:none;
-				}
-					
-				&::before{
-					content:"\f1ea";
-					font-family:FontAwesome;
-					display:flex;
-					flex-grow:0;
-					flex-shrink:0;
-					flex-basis:auto;
-					justify-content:center;
-					align-items:center;
-					float:left;
-					margin-right:10px;
-					padding:0;
-					height:2em;
-					width:2em;
-					border-radius:50%;
-					background-color:$black;
-					color: $light-gray;
-					font-size:2em;
-					line-height:1em;
-					/* other styles etc. */
-				}
-        	}
-    	}
+	&--light {
+		background-color:$white;	
 	}
+	
+	&__link {
+		font-size: .9em;
+		line-height: 1.4em;
+			&-other {   
+			display:flex;
+			align-items:top;
+			margin-top: 20px;
+			padding-top: calc-toRem(20);
 
+			&:link,&:visited,&:hover,&:active{
+			color:$color-campusGold;
+			text-decoration:none;
+			}
+
+			&::before{
+			content:"\f1ea";
+			font-family:FontAwesome;
+			display:flex;
+			flex-grow:0;
+			flex-shrink:0;
+			flex-basis:auto;
+			justify-content:center;
+			align-items:center;
+			float:left;
+			margin-right:calc-toRem(20);
+			margin-top: calc-toRem(-15);
+			padding:0;
+			height:2em;
+			width:2em;
+			border-radius:50%;
+			background-color:$black;
+			color: $light-gray;
+			font-size:2em;
+			line-height:1em;
+			}
+		}
+	}
+}
 
 .button {
 	margin-top: 1em !important;
 }
 
 .cta {
 	width: 100%;
-	padding: 1em;
+	padding-top: 20px;
+	padding-bottom: 30px;
 	background: repeating-linear-gradient(60deg,#707070,#707070 6px,#777777 6px,#777777 8px);
 	
-		&__title {
-			color: $white;
-			font-size: 1.5em;
-			line-height: 1.5em;
-			text-align: center;
-			padding: 0;
-		}
+	&__title {
+		color: $white;
+		font-size: 1.5em;
+		line-height: 1.5em;
+		text-align: center;
+		padding: 0;
+	}
 
+	&__description {
+		color: $white;
+		font-size: 1em;
+		line-height: 1em;
+		text-align: center;
+		padding-top: 0;
+		padding-bottom: 30px;
+	}
+}
+
+.btn-center {
```

Comment:
Unless this class is needed for something else, you could use the bootstrap helper class `block-center` instead https://getbootstrap.com/docs/3.3/css/#helper-classes-center